Sources/docc/DocCDocumentation.docc/adding-supplemental-content-to-a-documentation-catalog.md

```diff
@@ -89,10 +94,16 @@ Although writing documentation comments in source files has many benefits, in so
 * When your source documentation comments focus on the implementation of your
   code, and aren't appropriate for external documentation
 
-To add an extension file to your documentation catalog, create a file within the documentation catalog, then modify the first line of the file to identify the symbol that the file relates to using a symbol link in a level 1 header. 
-For more information on linking to symbols, see <doc:linking-to-symbols-and-other-content>.
+To add an extension file to your documentation catalog, create a file within the
+documentation catalog, then modify the first line of the file to identify the
+symbol that the file relates to using a symbol link in a level 1 header. For
+more information on linking to symbols, see
+<doc:linking-to-symbols-and-other-content>. DocC ignores file names of
+documentation extensions; you can choose any file name you would like as long as
+it uses an `.md` extension. DocC determines the URL path of source documentation
+from the symbol's name, type and parent type among other things.
 
-> Important: The symbol path for the page title of an extension file need to start with the name of a top-level symbol or the name of the framework.
+> Important: The symbol path for the page title of an extension file must start with the name of a top-level symbol or the name of the framework.
```

Comment:
It would be better to rephrase this sentence to explain the consequence of writing an incorrect symbol links. Maybe something like this:
```suggestion
> Important: 
> If DocC can't resolve the symbol link in the level 1 header of the extension file, it will raise a warning about the link and skip the rest of the content of that extension file. 
>
> DocC resolves symbol links in extension file headers relative to the module.
> To create an extension file for a nested type or member symbol, start the symbol link with the name of the top-level symbol that contains the nested type or member symbol.
> If you want you can prefix the symbol link with the name of the module but it's not necessary.
```

Reply:
Also, [this aside](https://github.com/apple/swift-docc/pull/874/files#diff-da6b9292790d774212f3cb84b563f00316f0a35879d71c639491910ee13d8782R158) contains mostly the same phrasing. It would be good to update both with the same information.